lib/pageflow/configuration/permissions.rb

```diff
@@ -0,0 +1,30 @@
+module Pageflow
+  class Configuration
+    # Permissions related options to be defined in the pageflow
+    # initializer of the main app.
+    class Permissions
+      def initialize(options = {})
+        @show_entry_features_to_non_admin =
+          options.fetch(:show_entry_features_to_non_admin, true)
+        @show_admin_boolean_to_non_admin =
+          options.fetch(:show_admin_boolean_to_non_admin, true)
+        @show_theming_dropdown_to_non_admin =
+          options.fetch(:show_theming_dropdown_to_non_admin, true)
+      end
+
+      # Allow non-admin users to see entry features. Defaults to true.
+      # @since edge
+      attr_accessor :show_entry_features_to_non_admin
+
+      # Allow non-admin users to know about admin boolean. Defaults to
+      # true.
+      # @since edge
+      attr_accessor :show_admin_boolean_to_non_admin
```

Comment:
I think it is appropriate. Maybe decoupling our naming scheme from a cancan-like syntax has its own benefits (such as thinking more freely about authorization), but I thought about this as well and am myself undecided